Extensions/xkit_preferences.js

```diff
@@ -394,7 +412,7 @@ XKit.extensions.xkit_preferences = new Object({
 
 		},
 
-		check: function(id) {
+		check_for_news_existing: function(id, version) {
```

Comment:
A comment header block before the function should be acceptable.  See the comment headers on the functions in the resourceBuilder files for an example.